src/utils/atoms/createStateMap.test.tsx

```
@@ -50,23 +50,22 @@ test('Replace init state with loading after delay', async () => {
+    { loadingStateDelayMs: 100 },
   );
 
-  render(
+  const { unmount } = render(
     <>
       {statesToComponents({
         init: <>Init</>,
         loading: <>Preloader</>,
         ready: () => <>Data</>,
       })}
     </>,
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)